### PR TITLE
Fixes Mobile view

### PR DESCRIPTION
**Pull request by adrijshikhar**
fixed responsiveness for mobile view
added hamburger in navbar for mobile view
added media-query mixins

### CODE REVIEW

File: css/styles.scss [hunk 1]
```diff
@@ -107,7 +122,16 @@ nav {
   width: 100%;
   box-sizing: border-box;
   z-index: 1;
-
+  input {
```
[the remark this answers is not in the document]
Comment (adrijshikhar) [reply css/styles.scss]: ok

File: css/styles.scss [hunk 2]
```diff
@@ -121,7 +145,86 @@ nav {
   .right {
     float: right;
     padding: 15px 0 0;
-  }
+  }@include sm {
```
[the remark this answers is not in the document]
Comment (adrijshikhar) [reply css/styles.scss]: ok

File: css/styles.scss [hunk 3]
```diff
@@ -121,7 +145,86 @@ nav {
...
+			#menu {
+				width: fit-content;
+				right: 0;
+				list-style-type: none;
+				-webkit-font-smoothing: antialiased;
+				max-height: 0;
+				overflow: hidden;
+				transition: all 0.5s ease-in-out;
+			}
+			input:checked ~ .ham > span {
```
[the remark this answers is not in the document]
Comment (adrijshikhar) [reply css/styles.scss]: ok

File: css/styles.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (pandeykartikey) [reply css/styles.scss]: you forgot to remove this.

[the remark this answers is not in the document]
Comment (adrijshikhar) [reply css/styles.scss]: done

**Comment by uttu357** [comment]
Rename the PR title to a proper title.

File: css/styles.scss [hunk 4]
```diff
@@ -27,7 +28,16 @@ body {
   min-height: 100%;
   font-family: AvenirNext;
 }
-
+h1 {
+	@include sm {
+		font-size: 1.8em;
+	}
```
Comment (uttu357) [inline css/styles.scss]: Indentation is all messed up. Won't merge before indentation is fixed.

File: css/styles.scss [hunk 5]
```diff
@@ -75,12 +85,17 @@ section.no-height {
 .bg-spl {
   position: relative;
   height: 100%;
+  @include sm() {
+		height: 70vh;
```
Comment (uttu357) [inline css/styles.scss]: Indentation is messed up.

File: css/styles.scss [hunk 6]
```diff
@@ -107,7 +122,16 @@ nav {
   width: 100%;
   box-sizing: border-box;
   z-index: 1;
-
+  #ham-input {
+		display: none;
+  }
+  	@include sm {
```
Comment (uttu357) [inline css/styles.scss]: Indentation is messed up.

**Comment by uttu357** [comment]
@adrijshikhar Fix indentation in the .scss file and then I can review it again. Until that follow the steps discussed on slack to provide a demo link.

File: css/styles.scss (hunk 4 above)
Comment (adrijshikhar) [reply css/styles.scss]: kp said not to indent as it would create many changes in the pr

**Comment by adrijshikhar** [comment]
kp said not to indent as it would create many changes in the pr

**Comment by pandeykartikey** [comment]
I said not to indent lines in which you made no changes as that increases the size of PR to Line changes ~1000. And make a different PR for the indentation issue.

**Comment by adrijshikhar** [comment]
Ohh, i get it

**Comment by kanav99** [comment]
LGTM. Good work.

**Pull request merged**